### ISSUE TITLE
Support multiple output files? 

### ISSUE THREAD

**Issue by MertNYuksel** [mataku/danger-ktlint#11]
Hello, 

We have multiple modules in our android app and we're using [ktlint-gradle](https://github.com/JLLeitschuh/ktlint-gradle). It creates multiple output files for each subproject in their build directory. You can change output directory but you can't aggregate generated reports. 

We want to use this danger plugin and with ktlint-gradle plugin but as far as i see, it's not possible use multiple report files. Is it possible to add a feature that gives support for multiple output files that matches with given regex?

**Comment by mataku** [comment]
Sorry for the late reply.

Currently, if you need an inline comment, you can do so below. 

```ruby
%w(report.json another_report.json).each do |report_file|
  ktlint.report_file = 'result.json'
  ktlint.lint
end
```


However, if not, I understand that it is inconvenient to have separate comments, so I will try to make it possible to output a combined result when the result files are specified by a regular expression or an array.

**Comment by mataku** [comment]
Released as v0.0.7.

You can specify like this.

```ruby
ktlint.skip_lint = true
ktlint.report_files_pattern = '**/report.json'  # => ['module_a/report.json',  'module_b/report.json', ...] 
ktlint.lint
```
